tests/test_daemon.cpp

```diff
@@ -592,18 +601,22 @@ TEST_P(DaemonCreateLaunchTestSuite, adds_ssh_keys_to_cloud_init_config)
             EXPECT_THAT(ssh_keys_stanza, YAMLNodeContainsSubString(expected_key));
         }));
 
-    send_command({GetParam()});
+    const auto [command, _] = GetParam();
+    send_command({command});
 }
 
 TEST_P(DaemonCreateLaunchTestSuite, adds_pollinate_user_agent_to_cloud_init_config)
 {
+    const auto [command, alias] = GetParam();
```

Comment:
also since this is the only place where the alias from `GetParam()` is used, wouldn't it be cleaner to just create a new `DaemonCreateLaunchAliasTestSuite` that implements `WithParamInterface<std::tuple<std::string, string>>` to be used here and to not modify all the other tests to ignore the alias?